src/Equinor.ProCoSys.IPO.Command/Validators/InvitationValidators/InvitationValidator.cs

```diff
@@ -178,6 +177,43 @@ public async Task<bool> ParticipantExistsAsync(int id, int invitationId, Cancell
                       where p.Id == id && EF.Property<int>(p, "InvitationId") == invitationId
                       select p).AnyAsync(cancellationToken);
 
+        public async Task<bool> IsSignedParticipantAsync(int id, int invitationId, CancellationToken cancellationToken)
+            => await (from p in _context.QuerySet<Participant>()
+                      where p.Id == id && EF.Property<int>(p, "InvitationId") == invitationId && p.SignedBy != null && p.SignedAtUtc != null
+                      select p).AnyAsync(cancellationToken);
+
+        public async Task<bool> HasOppositeAttendedStatusAsync(int id, int invitationId, bool attended, CancellationToken cancellationToken)
+            => await (from p in _context.QuerySet<Participant>()
+                where p.Id == id && EF.Property<int>(p, "InvitationId") == invitationId && p.Attended != attended
+                select p).AnyAsync(cancellationToken);
+
+        public async Task<bool> HasPermissionToEditParticipantAsync(int id, int invitationId, CancellationToken cancellationToken)
+        {
+            if (await InvitationHelper.HasIpoAdminPrivilege(_permissionCache, _plantProvider, _currentUserProvider)) { return true; }
```

Comment:
Format return in it's own line
![image](https://user-images.githubusercontent.com/19545528/159448812-b7f438ea-3513-4f11-9d4b-286171a024dd.png)

More of the same below